perception/tensorrt_yolox/src/tensorrt_yolox_node.cpp

```diff
@@ -32,23 +32,72 @@ TrtYoloXNode::TrtYoloXNode(const rclcpp::NodeOptions & node_options)
   using std::placeholders::_1;
   using std::chrono_literals::operator""ms;
 
-  std::string model_path = declare_parameter("model_path", "");
-  std::string label_path = declare_parameter("label_path", "");
-  std::string precision = declare_parameter("precision", "fp32");
-  // Objects with a score lower than this value will be ignored.
-  // This threshold will be ignored if specified model contains EfficientNMS_TRT module in it
-  float score_threshold = declare_parameter("score_threshold", 0.3);
-  // Detection results will be ignored if IoU over this value.
-  // This threshold will be ignored if specified model contains EfficientNMS_TRT module in it
-  float nms_threshold = declare_parameter("nms_threshold", 0.7);
+  auto declare_parameter_with_description =
+    [this](std::string name, auto default_val, std::string description = "") {
+      auto param_desc = rcl_interfaces::msg::ParameterDescriptor{};
+      param_desc.description = description;
+      return this->declare_parameter(name, default_val, param_desc);
+    };
+
+  std::string model_path = declare_parameter_with_description("model_path", "");
+  std::string label_path = declare_parameter_with_description("label_path", "");
```

Comment:
Is it necessary to use `declare_parameter_with_description` when the description is empty?

Reply:
@wep21 
I appreciate your suggestion. I just used `declare_parameter_with_description` for consistency, but as you said, it looks somewhat weird to use `..._with_description` without a description. So I added simple descriptions for these arguments at 7f7a9f81c671a61d1635c28183bb4ebc25f3f726